docker/dev/Dockerfile

```diff
@@ -1,18 +1,34 @@
-FROM openjdk:8-slim-buster
+FROM openjdk:8-alpine
```

Comment:
Same here, let's remove these changes and keep the PR focused on the remote kernel networking config

Reply:
Yes, I will clean up